models/issue.rb

```diff
@@ -36,6 +36,28 @@ def no_recent_activity
       @issue.updated_at < (Time.now - old_after_seconds)
    end
 
+   ##
+   # Returns true if an issue has a numeric label
+   def has_score
+      @issue.labels.any? do |label|
+         val = Integer(label.name) rescue nil
+         !!val
+      end
+   end
+
+   def has_pull
+      @issue.labels.any? { |label| label.name == 's-Under Review' }
+   end
+
+   def has_issue
+      keywords = ["close", "closes", "closed", "fix", "fixes", "fixed", "resolve", "resolves", "resolved", "description", "connect to", "connects to", "connected to", "connects"]
+
+      pattern = '\b' + '(' + keywords.join("|") + ')' + '\s+' + '#[0-9]+' + '\b'
+      re = Regexp.new(pattern, Regexp::IGNORECASE)
+
+      !(re =~ description).nil?
+   end
```

Comment:
What about
```ruby
keywords.any? do |kw|
   /\b #{kw} \s+ #[0-9]+ \b/ix =~ description
end
```
in place of lines 55-58? That keeps us to one string interpolation for the regex creation. The `x` on the regex is optional; it allows whitespace to be non-literal in the regex. If you think that's less readable, please don't switch. I'm really just bothered by the join (constructing regex like that feels weird).

Since nil is falsey, I think we can get away with `re =~ description` without the `nil?` and inversion.